servant-server/src/Servant/Server/Internal.hs

```diff
@@ -532,6 +532,16 @@ instance HasServer api context => HasServer (HttpVersion :> api) context where
   route Proxy context subserver =
     route (Proxy :: Proxy api) context (passToServer subserver httpVersion)
 
+data EmptyAPIServer = EmptyAPIServer
```

Comment:
we probably want `Eq` and `Show` instances (at least), `:<|>` have them too.

Same classes as in `EmptyAPI`?